pyproject.toml

```diff
@@ -80,6 +80,7 @@ flake8-rst-docstrings = "^0.0.12"
 flake8-executable = "^2.0"
 pep8-naming = "^0.9.1"
 darglint = "^1.1"
+libcst = "^0.3.1"
```

Comment:
One more thing: can you please move this dependency at the very top? Near `astor`

Reply:
done